tfjs-converter/python/requirements.txt

```diff
@@ -1,7 +1,6 @@
 flax>=0.6.2,<0.6.3
 jax>=0.3.16
 importlib_resources>=5.9.0
-protobuf<3.20,>=3.9.2
```

Comment:
Does TFJS converter require protobuf, or was this just to pin the version TensorFlow downloaded?

Reply:
It was just to pin the version due to cloud build failed to figure out the deps tree.